packages/@sanity/preview/src/constants.js

```diff
@@ -1,2 +1,4 @@
 export const INCLUDE_FIELDS_QUERY = ['_id', '_rev', '_type']
 export const INCLUDE_FIELDS = [...INCLUDE_FIELDS_QUERY, '_key']
+
+export const INVALID_PREVIEW_CONFIG = {}
```

Comment:
Should'nt this use `Symbol.for()` instead?

Reply:
That's probably better, yeah, thanks for pointing out :)